corehq/apps/export/utils.py

```diff
@@ -50,7 +50,10 @@ def convert_saved_export_to_export_instance(saved_export):
                     column_index=column.index,
                 )
 
-            new_column = new_table.get_column(_convert_index_to_path(index))
+            new_column = new_table.get_column(
+                _convert_index_to_path(index),
```

Comment:
Yeah, so `ExportItem` paths are lists of strings, but `TableConfiguration` and `ExportGroupSchema` paths are lists of `PathNode`s. I did it that way because `ExportItems` don't need to know which parts of their paths are repeats. But maybe it would just be overall easier if all the paths were the same format.